### DISCUSSION TITLE
Ajax Search - Autocomplete & Suggestions

### DISCUSSION THREAD

**Discussion by osman-kocak** [inspireui/support#5703] [New Feature/Ideas Requests]
I can please support the feature so that the customer can see the suggestions while searching.

<img width="834" alt="Screenshot 2023-01-10 at 23 58 28" src="https://user-images.githubusercontent.com/57155882/211671110-b52a51ec-cef5-4fff-ae2b-b7c7fcb31690.png">

**Comment by tryabp** [comment]
this is must have feature!

**Comment by osman-kocak** [comment]
up

**Comment by osman-kocak** [comment]
@minhcasi @luyx2412
